alpaca_trade_api/rest.py

```diff
@@ -612,29 +612,38 @@ def _data_get(self,
                   symbol_or_symbols: Union[str, List[str]],
                   api_version: str = 'v2',
                   endpoint_base: str = 'stocks',
+                  resp_grouped_by_symbol: bool = None,
```

Comment:
[minor] the type hint should be `Optional[None]`

Reply:
I assumed you meant `Optional[bool]`. Changed to that instead.